lib/commands/command-info.bash

```diff
@@ -2,9 +2,21 @@
 # shellcheck source=lib/functions/plugins.bash
 . "$(dirname "$(dirname "$0")")/lib/functions/plugins.bash"
 
+get_shell() {
+  if [[ -z $FISH_VERSION ]]; then
+    echo "fish"
+  elif [[ -z $ZSH_VERSION ]]; then
+    echo "zsh"
+  elif [[ -z $BASH_VERSION ]]; then
+    echo "bash"
+  else
+    echo $SHELL --version
+  fi
+}
+
 info_command() {
   printf "%s:\\n%s\\n\\n" "OS" "$(uname -a)"
-  printf "%s:\\n%s\\n\\n" "SHELL" "$($SHELL --version)"
+  printf "%s:\\n%s\\n\\n" "SHELL" "$(get_shell)"
```

Comment:
I think we may want to report both `$SHELL --version` and whatever shell and version is computed by your `get_shell` function above.

Reply:
sure thing, i've updated the output in [91527a3](https://github.com/asdf-vm/asdf/pull/1217/commits/91527a3df04b44a69df9de8a9789920ef42ecb0b)

Reply:
Sorry, I should have been more specific in my comment. Something like this:

```
...
printf "%s:\\n%s\\n\\n" "SHELL" "$($SHELL --version)"
printf "%s:\\n%s\\n\\n" "COMPUTED_SHELL" "$(get_shell)"
...
```

So that we print both `$SHELL --version` and whatever your `get_shell` function determines side by side. Sometimes the values will match, sometime they won't.

Reply:
i'm happy to add this however, i'm unsure what you'll actually get from it, except for confusion. take my example, I don't use zsh (it is on Mac by default) yet, the variable was incorrectly suggesting it was. asdf isn't installed in zsh nor configured so it is a red herring for any diagnosis related. 

is there a particular reason you'd like this to stick around?

Reply:
So my thinking was this if we have two imperfect ways of determining the current shell, why not just print both? If they are the same, it's confirmation that they both are correct. If they differ, we can inquire as to why.

Reply:
gotcha, so the ordering of the `if/else` _should_ prevent the imperfect approach as if the variable is found, the shell is actually in use. if none of those are found, fallback to the best effort `$SHELL --version`. i've tested this on fish, zsh, bash and elvish locally and all of them returned the correct shell each time instead of falling back to the `$SHELL --version` that reports incorrectly.